Find a functionally equivalent code:android.print.PrintDocumentInfo.Builder.build:COMMENT
Method Modifier: public      
Comment:/**
 * Creates a new {@link PrintDocumentInfo} instance.
 *
 * @return The new instance.
 */

@@ -1,3 +1,7 @@
 {
+    // wiritten PDF file for the page count.
+    if (mPrototype.mPageCount == 0) {
+        mPrototype.mPageCount = PAGE_COUNT_UNKNOWN;
+    }
     return new PrintDocumentInfo(mPrototype);
 }

